sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/CommandHandlerProvider.java

```diff
@@ -38,8 +39,18 @@
     public Map<String, CommandHandler> namedHandlers() {
         Map<String, CommandHandler> map = new HashMap<String, CommandHandler>();
         List<CommandHandler> handlers = spiLoader.loadInstanceList();
+        List<CommandHandlerInterceptor> commandHandlerInterceptors = SpiLoader.of(CommandHandlerInterceptor.class).loadInstanceListSorted();
         for (CommandHandler handler : handlers) {
             String name = parseCommandName(handler);
+            if (!commandHandlerInterceptors.isEmpty()) {
+                List<CommandHandlerInterceptor> interceptors = new ArrayList<>();
+                for (CommandHandlerInterceptor commandHandlerInterceptor : commandHandlerInterceptors) {
+                    if (commandHandlerInterceptor.shouldIntercept(name)) {
+                        interceptors.add(commandHandlerInterceptor);
+                    }
+                }
+               handler = new InterceptingCommandHandler(handler, interceptors);
+            }
             if (!StringUtil.isEmpty(name)) {
```

Comment:
I think we can judge the name first. if name is empty, skip it quickly.

Reply:
> I think we can judge the name first. if name is empty, skip it quickly.

this is a good suggestion